setup.py

```diff
@@ -35,60 +35,70 @@
         libraries = ["arb", "flint", "mpir", "mpfr", "pthreads"]
 else:
     # On Ubuntu libarb.so is called libflint-arb.so
-    if os.getenv('PYTHON_FLINT_LIBFLINT_ARB'):
-        arb = 'flint-arb'
+    if os.getenv("PYTHON_FLINT_LIBFLINT_ARB"):
+        arb = "flint-arb"
     else:
-        arb = 'arb'
+        arb = "arb"
 
     libraries = [arb, "flint"]
-    (opt,) = get_config_vars('OPT')
-    os.environ['OPT'] = " ".join(flag for flag in opt.split() if flag != '-Wstrict-prototypes')
+    (opt,) = get_config_vars("OPT")
+    os.environ["OPT"] = " ".join(
+        flag for flag in opt.split() if flag != "-Wstrict-prototypes"
+    )
```

Comment:
Your editor is doing lots of strange things now.

Reply:
Oh yeah, I have some automated linting with `black` configured from a previous project I was doing where I got lazy and found some tricks to automate formatting while collaborating with a bunch of people. I'll remove it for this, just so used to everything being modified I didn't notice.

Reply:
Reverted the formatting back